.travis.yml

```
@@ -8,8 +8,8 @@ env:
 
 matrix:
   include:
-  - go: 1.12.x
-  - go: 1.13.x
+  - go: 1.14.x
+  - go: 1.15.x
```

we can use `stable` and `oldstable` here, which will always point to the currently supported versions.

https://github.com/travis-ci/gimme/releases/tag/v1.5.4

```suggestion
  - go: oldstable
  - go: stable
```